backend/server/http/api/handlers/floorplans/handlers.go

```diff
@@ -7,10 +7,11 @@ import (
 
 	"github.com/gorilla/mux"
 
-	"github.com/lanets/floorplanets/backend/api/handlers/decorators"
+	"github.com/lanets/floorplanets/backend/server/http/api/handlers/decorators"
+	"github.com/lanets/floorplanets/backend/server/http/api/server"
 )
 
-func floorplansGetHandler() http.Handler {
+func floorplansGetHandler(server server.APIServer) http.Handler {
 
 	handler := func(w http.ResponseWriter, r *http.Request) {
 		// Bogus response for now :)
```

Comment:
It's nice that the `/floorplans/1` floorplan is hardcoded, but don't you think we should also add it to the list returned via
the `/floorplans/` endpoint ? Because right now I get an empty list when hitting the `/floorplans/` endpoint.